tests/test_project.py

```diff
@@ -417,6 +417,65 @@ def updater(remotes):
     assert ur.tr_head_0 == v1_0
     assert ur.tr_head_1 == v2_0
 
+def test_update_head_0(west_init_tmpdir):
+    # Verify that using HEAD~0 as the revision causes west to not touch the
+    # local repo. In this case, local zephyr should remain unchanged, even
+    # though it is referenced in the new top-level manifest 'my_manifest',
+    # since it's referenced with HEAD~0. Check that a local commit remains, and
+    # that local changes are kept as is. An invalid url ('na') is used to check
+    # that west doesn't attempt to access it.
+    # Expect only Kconfiglib to be updated, since it's mentioned in the new
+    # manifest's name-allowlist.
```

Comment:
I personally don't see a reason to do that since these aren't API functions.

Reply:
I frequently ask Doxygen to produce call graphs and other documentation for **internal** functions, not just for APIs. It's a good code indexer and browser.

I don't see why the same should not be possible in Python.

Reply:
If you have a legit use case for this and it's not just personal preference, I would merge a PR that converted these all to use docstrings instead of doc comments, but you would need to write it!

Reply:
OK forget it, I understand your preference for consistency over a progressive transition.